scripts/cloud-hooks/hooks/common/post-db-copy/db-scrub.sh

```diff
@@ -18,6 +18,6 @@ repo_root="/var/www/html/$site.$target_env"
 export PATH=$repo_root/vendor/bin:$PATH
 cd $repo_root
 
-blt artifact:ac-hooks:db-scrub $site $target_env $db_name $source_env
+blt artifact:ac-hooks:db-scrub $site $target_env $db_name $source_env --ansi=false
```

Comment:
is there a reason this one is not drush.asni?

Reply:
Yeah, the reason is this was my last PR of the day and I must have been getting tired :) I'll fix it.